.rusty-hook.toml

```diff
@@ -0,0 +1,5 @@
+[hooks]
+pre-commit = "cargo fmt --all -- --check"
```

Comment:
No certain how `rusty-hook` works. If it behaves quite similar to pre-commit hook, I think it's a nice idea to run clips as well. What do you think? Although it will take a significant time.